lib/medera.ex

```diff
@@ -39,4 +28,30 @@ defmodule Medera do
     Endpoint.config_change(changed, removed)
     :ok
   end
+
+  def child_specs(true) do
+    slack_children() ++ web_children() ++ minion_children()
```

Comment:
Are there any applications that are started, but have no use in some situations?